### ISSUE TITLE
AbstractLAPACK dgeev argument checking

### ISSUE THREAD

**Issue by uhoefel** [luhenry/netlib#12]
There seems to be an issue in `AbstractLAPACK.java` in `dgeev`.
My test input works with `netlib.fommil` and looks like this:

```java
int n = 2;
double[] a = {1.0, 0.75, 0.5, 1.0};
double[] wr = new double[2];
double[] wi = new double[2];
int lwork = 68; // optimal size from previous query
double[] work = new double[lwork];
intW info = new intW(2);

LAPACK.getInstance().dgeev("N", "N", n, a, n, wr, wi, new double[1], 1, new double[1], 1, work, lwork, info);
```
To my understanding, that should give the following eigenvalues in `wr`: 

$$\lambda_{1,2} = \frac{\pm\sqrt{6}+4}{4}$$

My suspicion is that lines 338-341
```java
if (lsame("N", jobvl))
    checkIndex(offsetvl + n * ldvl - 1, vl.length);
if (lsame("N", jobvr))
    checkIndex(offsetvr + n * ldvr - 1, vr.length);
```
should read 
```java
if (lsame("V", jobvl))
    checkIndex(offsetvl + n * ldvl - 1, vl.length);
if (lsame("V", jobvr))
    checkIndex(offsetvr + n * ldvr - 1, vr.length);
```
The [documentation](http://www.netlib.org/lapack/explore-html/d9/d8e/group__double_g_eeigen_ga66e19253344358f5dee1e60502b9e96f.html) of LAPACK states for `vl` that (similar for `vr`)
> If JOBVL = 'N', VL is not referenced.

so I would at least not intuit that its length matters (or I may very well miss something).

PS: Thanks for fixing the split packages!

**Comment by uhoefel** [comment]
Seems correct as far as I can judge. I think the checks some lines above (e.g. 4962-4965, but also in dgeev) probably should be flipped to `"V"` as well:

```java
    if (lsame("N", jobvl))
      requireNonNull(vl);
    if (lsame("N", jobvr))
      requireNonNull(vr);
```
--> 
```java
    if (lsame("V", jobvl))
      requireNonNull(vl);
    if (lsame("V", jobvr))
      requireNonNull(vr);
```

**Comment by luhenry** [comment]
It's been released in [v3.0.2](https://github.com/luhenry/netlib/releases/tag/v3.0.2)
